csl_demo/BlockfrostProvider.gd

```diff
@@ -18,37 +18,37 @@ class Request:
 class ProtocolParametersRequest extends Request:
 	var epoch: int = 0
 	
-	func _init(epoch: int):
-		self.epoch = epoch
+	func _init(epoch_: int) -> void:
+		self.epoch = epoch_
 	
-	func url():
-		return "epochs/%s/parameters" % (self.epoch if self.epoch != 0 else "latest")
+	func url() -> String:
+		return "epochs/%s/parameters" % (str(self.epoch) if self.epoch != 0 else "latest")
```

Comment:
At first I thought the `str()` coercion is there because otherwise a warning is emitted due to the two if branches having different types. But removing it doesn't seem to change anything, so this is redundant indeed.

Reply:
It seems like the type annotation prevents the warning.